Assets/Decontainment/Bot/Weapons/Projectiles/Scripts/ShotgunPellet.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Bot;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShotgunPellet : Projectile {
+
+    [SerializeField]
+    private float lifeTimer;
```

Comment:
Give `lifeTimer` a default value. Otherwise this annoying warning shows up in the log every time the code compiles.